addon/mixins/table-header.js

```diff
@@ -66,13 +67,22 @@ export default Ember.Mixin.create({
   resizeOnDrag: false,
 
   /**
+   * CSS classes to be applied to an `<i class="lt-sort-icon></i>` tag that is
+   * inserted into the column's `<th>` element.
+   *
+   * For instance, if you have installed `ember-font-awesome`, you can set
+   * `iconAscending` to `'fa fa-sort-asc'`, which would yield this markup:
+   * `<i class="lt-sort-icon fa fa-sort-asc"></i>`
+   *
```

Comment:
In accordance with https://github.com/offirgolan/ember-light-table/pull/438#discussion_r123255492 this might be better as well?

```js
  /**
   * CSS classes to be applied to an `<i class="lt-sort-icon></i>` tag that is
   * inserted into the column's `<th>` element.
   *
   * For instance, if you have installed `ember-font-awesome` or include the
   * `font-awesome` assets manually (e.g. via a CDN), you can set
   * `iconAscending` to `'fa fa-sort-asc'`, which would yield this markup:
   * `<i class="lt-sort-icon fa fa-sort-asc"></i>`
   *
   * @property iconAscending
   * @type {String}
   * @default ''
   */
```